src/i18n.js

```diff
@@ -3,22 +3,37 @@ import ICU from 'i18next-icu'
 import XHR from 'i18next-xhr-backend'
 import LanguageDetector from 'i18next-browser-languagedetector'
 
+import cs from 'i18next-icu/locale-data/cs'
+import da from 'i18next-icu/locale-data/da'
+import de from 'i18next-icu/locale-data/de'
 import en from 'i18next-icu/locale-data/en'
+import es from 'i18next-icu/locale-data/es'
+import fr from 'i18next-icu/locale-data/fr'
+import it from 'i18next-icu/locale-data/it'
+import ja from 'i18next-icu/locale-data/ja'
 import ko from 'i18next-icu/locale-data/ko'
+import nl from 'i18next-icu/locale-data/nl'
 import no from 'i18next-icu/locale-data/no'
 import pl from 'i18next-icu/locale-data/pl'
 import pt from 'i18next-icu/locale-data/pt'
+import ru from 'i18next-icu/locale-data/ru'
 import sv from 'i18next-icu/locale-data/sv'
+import zh from 'i18next-icu/locale-data/zh'
+
+const localeData = [cs, da, de, en, es, fr, it, ja, ko, nl, no, pl, pt, ru, sv, zh]
 
 i18n
-  .use(new ICU({
-    localeData: [en, ko, no, pl, pt, sv]
-  }))
+  .use(new ICU({ localeData }))
   .use(XHR)
   .use(LanguageDetector)
   .init({
     ns: ['explore'],
-    fallbackLng: 'en',
+    fallbackLng: {
+      'zh-Hans': ['zh-CN', 'en'],
+      'zh-Hant': ['zh-TW', 'en'],
+      zh: ['zh-CN', 'en'],
+      default: ['en']
+    },
```

Comment:
`zh` locale is tricky, this detects country-agnostic codes which some users may use and maps them to correct version

Reply:
AHA! Thank you for finding that!